src/20_Components/amp-bind.html

```diff
@@ -79,6 +88,25 @@
 
   `[property]="expression"`
 
+-->
+
+<!-- #### Text for textContent -->
+<!--
+The text of this &lt;p&gt; will change when the `currentAnimal` variable
+changes.
+-->
+<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+
+<!-- #### class for CSS classes -->
+<!--
+The class of this &lt;p&gt; will change when
+either the `allAnimals` or `currentAnimal` implicit state variables change.
+-->
+
+<p [class]="allAnimals[currentAnimal].styleClass">This text will have have a red background color when cat is selected.</p>
```

Comment:
s/This text will have have a red background color when cat is selected./Each animal has a different background color./